### PR TITLE
LPS-157270 buildLang

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout/look_and_feel/GlobalCETOrderHelpIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			<div>{Liferay.Language.get('order')}</div>
+
+			<ClayPopover
+				alignPosition="top"
+				closeOnClickOutside
+				header={title}
+				onShowChange={setShow}
+				show={show}
+				trigger={
+					<ClayButtonWithIcon
```
Comment (victorg1991) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout/look_and_feel/GlobalCETOrderHelpIcon.tsx:33]: I think we can use this pattern here: https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/ :3

Comment (victorg1991) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/layout/look_and_feel/GlobalCETOrderHelpIcon.tsx:33]: Also should this popover be displayed on hover?

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:close

**Pull request closed without merging**